### PR TITLE
[Easy] fix event receiver

### PR DESCRIPTION
**Pull request by fleupold**
My refactoring broke the event receiver (which I discovered by running the e2e tests). This PR should fix the issues introduced.

### CODE REVIEW

**Comment by bh2smith** [comment]
My only question is what do you expect to happen when the values inside this dict are not of integer type.

File: event_listener/dfusion_db/models.py [hunk 1]
```diff
@@ -37,16 +37,15 @@ class Deposit(NamedTuple):
 
     @classmethod
     def from_dictionary(cls, data: Dict[str, Any]) -> "Deposit":
-        assert data.keys() == {'accountId', 'tokenId', 'amount',
-                               'slot', 'slotIndex'}, "Unexpected Event Keys"
-        assert all(isinstance(val, int) for val in data.values()
```
Comment (bh2smith) [inline event_listener/dfusion_db/models.py]: I see this assertion has been removed...

File: event_listener/dfusion_db/snapp_event_receiver.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def save(self, event: Dict[str, Any], block_info: Dict[str, Any]) -> None:
 
         try:
             self.initialize_accounts(
-                event['maxTokens'], event['numAccounts'], state_hash)
+                event['maxTokens'], event['maxAccounts'], state_hash)
```
Comment (bh2smith) [inline event_listener/dfusion_db/snapp_event_receiver.py]: This looks like the reason something was breaking.

**Comment by fleupold** [comment]
> My only question is what do you expect to happen when the values inside this dict are not of integer type.

We still check this since `int()` will throw if it's argument cannot be converted to integer. I can add a unit test.

**Comment by bh2smith** [comment]
Seems good to me. Feel free to merge!

**Pull request merged**